source/jquery.flot.touchNavigate.js

```diff
@@ -127,6 +151,10 @@
 
         doubleTap = {
             recenterPlot: function(e) {
+                if (e && e.details && e.details.type === 'touchMove') {
+                    // do not recenter during touch moving;
+                    return;
+                }
```

Comment:
Some time the graph would recenter when panning the graph. I think it is not expected behavior. The root cause is that the `doubletap` custom event is triggered for `touchMove`.

Adding an event filter here can prevent recentering on touch panning.

Reply:
![unexpected recenter](https://user-images.githubusercontent.com/39820924/51817741-f47b2d80-2306-11e9-8f14-87b28ede2b14.gif)
This gif is captured with master branch example.